### PR TITLE
Quick wins

### PR DESCRIPTION
**Pull request by bastianallgeier**
## Describe the PR
This PR closes three simple issues at once. I combined them although they are not related at all. But the fixes are so simple, that separate PRs felt too much.

- #2856
- #3140

## Ready?

- [ ] Unit tests for fixed bug/feature
- [ ] In-code documentation (wherever needed)
- [ ] CI checks pass

## When merging

- [ ] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
- [x] Add changes to release notes draft in Notion

### CODE REVIEW

**Comment by bastianallgeier** [comment]
@getkirby/kirby-staff help. Codecov has a new interface again, or something's broken. Do you know how to see now where the missing coverage is?

File: src/Database/Db.php [hunk 1]
```diff
@@ -190,7 +190,7 @@ public static function __callStatic(string $method, $arguments)
  * @param array $values An array of values which should be inserted
  * @return int ID of the inserted row
  */
-Db::$queries['insert'] = function (string $table, array $values): int {
+Db::$queries['insert'] = function (string $table, array $values) {
```
Comment (afbora) [inline src/Database/Db.php:191]: This line uncovered.

Comment (bastianallgeier) [reply src/Database/Db.php:191]: I don't really understand why there's a DbTest::testInsert method with a @coversNothing flag. In theory the line is definitely covered.

Comment (afbora) [reply src/Database/Db.php:191]: I guess we need `@codeCoverageIgnoreStart` and `@codeCoverageIgnoreEnd` for inline codes 🤔

Comment (distantnative) [reply src/Database/Db.php:191]: I think Ahmet is right, the `@codeCoverageIgnore` seems to not be applied at all to these inline closures, @lukasbestle .

**Comment by distantnative** [comment]
@bastianallgeier I think removing the default `autocomplete=off` and `spellcheck=off` needs to be considered a breaking change as it can result in quite a different user experience using the Panel if the developer just upgrades and doesn't set it up according to their needs.

**Comment by bastianallgeier** [comment]
@distantnative you are right. I haven't thought about it this way.

**Comment by bastianallgeier** [comment]
I've reverted the TextInput changes

**Review by distantnative — APPROVED** [review]

**Pull request merged**